src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1469,4 +1469,21 @@ public static int calculateGQFromPLs(final int[] plValues) {
         }
         return second - first;
     }
+
+    /**
+     *  Create a string with existing filters plus the one to append
+     * @param vc VariantContext to base initial filter values.  Not {@code null}
+     * @param filterToAppend the filter value to append to the strings Not {@code null}
+     * @return String of filter values.  Sorted alphabetically.
+     */
+    public static List<String> createFilterListWithAppend(VariantContext vc, String filterToAppend) {
```

Comment:
Then it can just be appending to a list and sorting.  And this is in GATKVariantContextUtils, after all.

I liked the convenience of just being able to get an updated filter set in one call.

No action, unless you strongly disagree.  

No action.